### ISSUE TITLE
Default Emojis Messed Up

### ISSUE THREAD

**Issue by GameShadows** [AnemoneTeam/Anemone#139]
iPhone SE
iOS 9.3.3
I had BytaFont 3 before, but I never updated to their new theming method and I restored the emojis via bytafont backup, and then uninstalled BytaFont 3 and finally installed all 3 Emoji themes from mwoolweaver's repo, I noticed in safe mode that the themed emojis were not matching the default emojis (The emojis were still themed on my keyboard in safe mode, but when I tapped them they displayed in default emoji theme). However, upon checking with my friend the themed emojis were right and it appears the default emoji font has the images on the wrong emojis. So for instance the kissing face is actually the crying laughing face.

**Comment by coolstar** [comment]
Are emojis messed up if you disable all fonts in Anemone?

**Comment by GameShadows** [comment]
Yes. That's how I confirmed it was the default setup, the themed ones were
correct and when I turned off the themes the stock emojis were wrong.

**Comment by coolstar** [comment]
BytaFont has screwed up your system files. You'll need to find original copies of the default Emoji fonts and place them in manually.

**Comment by mwoolweaver** [comment]
Try using this with BytaFont and see if it will fix it...

[Original Fonts iOS 9.3.x - Emoji - Original iOS 9.3.x - Emoji](http://cydia.saurik.com/package/com.bytafont3.originalios93xemoji/)
